arrows/vxl/aligned_edge_detection.h

```diff
@@ -0,0 +1,55 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef KWIVER_ARROWS_VXL_ALIGNED_EDGE_DETECTION_
+#define KWIVER_ARROWS_VXL_ALIGNED_EDGE_DETECTION_
+
+#include <arrows/vxl/kwiver_algo_vxl_export.h>
+
+#include <vital/algo/image_filter.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
+/**
+ * @brief Extract axis-aligned edges
+ */
```

Comment:
Please follow [the style guide](https://github.com/Kitware/kwiver/blob/master/doc/code-style.rst#api-documentation-doc):
```suggestion
/// Extract axis-aligned edges.
```

The rest of the doc looks mostly okay besides missing periods.